lucene/directory-provider/src/test/java/org/infinispan/hibernate/search/TwoNodesTest.java

```diff
@@ -60,6 +60,7 @@ public void testSomething() {
       } finally {
          nodeC.close();
       }
+      waitMembersCount(nodea, SimpleEmail.class, 2);
```

Comment:
Actually about that if you look closer `loopCounter` is never incremented.  I was originally saying this because a time based solution is better, but if you want just changing it to do a post increment I would be fine with that as well.